binding.gyp

```diff
@@ -2,20 +2,25 @@
   "targets": [
     {
       "target_name": "deltachat",
+      "conditions": [
+        [ "OS == 'win'", {}],
```

Comment:
Here's a seam where we should introduce e.g. `deltachat.lib` or whatever comes out of the windows build of `deltachta-core`.